redis-voting-app/main.star

```diff
@@ -0,0 +1,23 @@
+main_redis_module = import_module("github.com/kurtosis-tech/redis-package/main.star")
+
+VOTING_APP_IMAGE = "mcr.microsoft.com/azuredocs/azure-vote-front:v1"
+
+def run(plan, args):
+
+    plan.print("Spinning up the Redis Package")
+    redis_run_output = main_redis_module.run(plan, args)
+
+    redis_service_name = redis_run_output["hostname"]
```

Comment:
shouldn't be `redis_hostname` or `redis_address` the var name ?

Reply:
You are right; I was using service name earlier from output but then switched to hostname to show a feature + for it to work in k8s! 👍